packages/connection-model/lib/connect.js

```diff
@@ -59,7 +66,7 @@ async function connect(model, setupListeners) {
 
   debug('connecting ...');
 
-  const url = model.driverUrlWithSsh;
+  const url = removeGssapiServiceName(model.driverUrlWithSsh);
```

Comment:
Maybe I'm missing something, but I'm not sure why we need to do this explicitly here like that. `connection-model` removes/ignores all options that are not specified in the model definition, so if this is a deprecated option that we don't need anymore, shouldn't we just remove it from the model definition here?

https://github.com/mongodb-js/compass/blob/8836f408906058e65310af44678dd26391a2f87c/packages/connection-model/lib/model.js#L124

Reply:
Tried that, it breaks the form service name field .. is a temporary workaround anyway, that's not really a deprecated option but a deprecated query param, so we will have to wire it properly after the driver releases a fix.

Reply:
Ahhh, got it! I knew that it couldn't be so easy 😆